packages/edit-post/src/components/text-editor/style.scss

```diff
@@ -18,7 +18,7 @@
 
 		// Always show outlines in code editor
 		.editor-post-title__input {
-			border: $border-width solid $gray-400;
+			border: $border-width solid $gray-700;
```

Comment:
The border doesn't need to meet more than 3:1 ratio (UI and large text), and by having a lighter gray it provides contrast between neutral and focused state.

Would appreciate that changed to $gray-600 before merging.

Reply:
Agreed the borders need only a 3:1 contrast. Also mentioned in the issue:
>  it should be at least 3:1 (non-text contrast).

The placeholder text still requires a 4.5:1 contrast ratio.

Reply:
Alright, I've changed the unselected border colors to `$gray-600` and will merge when tests pass.

![image](https://user-images.githubusercontent.com/19592990/94155518-a5233280-fe44-11ea-90ee-7a21be48cd93.png)